mmengine/hooks/ema_hook.py

```diff
@@ -32,6 +33,13 @@ class EMAHook(Hook):
             Defaults to 0.
         begin_epoch (int): The number of epoch to enable ``EMAHook``. Defaults
             to 0.
+        revise_keys (list): A list of customized keywords to modify the
+                state_dict in checkpoint. Each item is a (pattern, replacement)
+                pair of the regular expression operations. Default: strip
+                the prefix 'module.' by [(r'^module\\.', '')].
+        resume (bool): Whether to resume ema model. If ``resume`` is True and
```

Comment:
Whether to resume is best judged from the `_resume` attribute of the runner. Adding the `resume` parameter to the EMAHook will make it more difficult to use.